group_vars/calculon_cluster/ip_addresses.yml

```diff
@@ -170,4 +158,16 @@ ip_addresses:
     vlan: internal_management
     fqdn:
     desc: Jumphost
+  gattaca01:
```

Comment:
These gattaca machines are not part of the Calculon cluster. It is better to add them to 
```group_vars/all/ip_addresses.yml```
In ```group_vars/calculon_cluster/vars.yml ``` you should move them then from group `calculon_cluster` to group `all` in the list of `additional_etc_hosts`.

Reply:
You're right, fixed it